src/server/pps/server/api_server_test.go

```diff
@@ -350,3 +353,62 @@ func TestUpdatePipelineInputBranch(t *testing.T) {
 		true, /* update */
 	))
 }
+
+func TestGetClusterDefaults(t *testing.T) {
```

Comment:
At some point there needs to be a `server_test` that starts up pachyderm with the environment variables and ensures that they are moved to cluster defaults.